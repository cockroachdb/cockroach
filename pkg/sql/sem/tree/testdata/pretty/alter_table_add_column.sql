ALTER TABLE foo ADD COLUMN "baR" "Mixed Case"
  Default ('foo'::"BarBazBaq") ON UPDATe ('baz':::"With Emojis too 👍"), ADD COLUMN "baR" "DefaultDB"."👎"."Mixed Case"
  Default ('foo'::"DefaultDB".public."Mixed Case") ON UPDATe ('baz':::"👎"."Mixed Case")
