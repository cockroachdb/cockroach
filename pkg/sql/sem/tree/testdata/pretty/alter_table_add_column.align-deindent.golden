// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
ALTER TABLE foo
	ADD COLUMN
		"baR"
			"Mixed Case"
			DEFAULT (
				'foo'::"BarBazBaq"
			)
			ON UPDATE (
				'baz':::"With Emojis too üëç"
			),
	ADD COLUMN
		"baR"
			"DefaultDB".üëé."Mixed Case"
			DEFAULT (
				'foo'::"DefaultDB".public."Mixed Case"
			)
			ON UPDATE (
				'baz':::üëé."Mixed Case"
			)

20:
--------------------
ALTER TABLE foo
	ADD COLUMN "baR"
				"Mixed Case"
				DEFAULT (
					'foo'::"BarBazBaq"
				)
				ON UPDATE (
					'baz':::"With Emojis too üëç"
				),
	ADD COLUMN "baR"
				"DefaultDB".üëé."Mixed Case"
				DEFAULT (
					'foo'::"DefaultDB".public."Mixed Case"
				)
				ON UPDATE (
					'baz':::üëé."Mixed Case"
				)

26:
--------------------------
ALTER TABLE foo ADD COLUMN
					"baR"
						"Mixed Case"
						DEFAULT (
							'foo'::"BarBazBaq"
						)
						ON UPDATE (
							'baz':::"With Emojis too üëç"
						),
                ADD COLUMN
					"baR"
						"DefaultDB".üëé."Mixed Case"
						DEFAULT (
							'foo'::"DefaultDB".public."Mixed Case"
						)
						ON UPDATE (
							'baz':::üëé."Mixed Case"
						)

32:
--------------------------------
ALTER TABLE foo ADD COLUMN "baR"
							"Mixed Case"
							DEFAULT (
								'foo'::"BarBazBaq"
							)
							ON UPDATE (
								'baz':::"With Emojis too üëç"
							),
                ADD COLUMN "baR"
							"DefaultDB".üëé."Mixed Case"
							DEFAULT (
								'foo'::"DefaultDB".public."Mixed Case"
							)
							ON UPDATE (
								'baz':::üëé."Mixed Case"
							)

45:
---------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case"
                                 DEFAULT (
									'foo'::"BarBazBaq"
                                 )
                                 ON UPDATE (
									'baz':::"With Emojis too üëç"
                                 ),
                ADD COLUMN "baR"
							"DefaultDB".üëé."Mixed Case"
							DEFAULT (
								'foo'::"DefaultDB".public."Mixed Case"
							)
							ON UPDATE (
								'baz':::üëé."Mixed Case"
							)

61:
-------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case"
                                 DEFAULT ('foo'::"BarBazBaq")
                                 ON UPDATE (
									'baz':::"With Emojis too üëç"
                                 ),
                ADD COLUMN "baR"
							"DefaultDB".üëé."Mixed Case"
							DEFAULT (
								'foo'::"DefaultDB".public."Mixed Case"
							)
							ON UPDATE (
								'baz':::üëé."Mixed Case"
							)

62:
--------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case"
                                 DEFAULT ('foo'::"BarBazBaq")
                                 ON UPDATE (
									'baz':::"With Emojis too üëç"
                                 ),
                ADD COLUMN "baR" "DefaultDB".üëé."Mixed Case"
                                 DEFAULT (
									'foo'::"DefaultDB".public."Mixed Case"
                                 )
                                 ON UPDATE (
									'baz':::üëé."Mixed Case"
                                 )

70:
----------------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case"
                                 DEFAULT ('foo'::"BarBazBaq")
                                 ON UPDATE (
									'baz':::"With Emojis too üëç"
                                 ),
                ADD COLUMN "baR" "DefaultDB".üëé."Mixed Case"
                                 DEFAULT (
									'foo'::"DefaultDB".public."Mixed Case"
                                 )
                                 ON UPDATE ('baz':::üëé."Mixed Case")

76:
----------------------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case"
                                 DEFAULT ('foo'::"BarBazBaq")
                                 ON UPDATE ('baz':::"With Emojis too üëç"),
                ADD COLUMN "baR" "DefaultDB".üëé."Mixed Case"
                                 DEFAULT (
									'foo'::"DefaultDB".public."Mixed Case"
                                 )
                                 ON UPDATE ('baz':::üëé."Mixed Case")

81:
---------------------------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case"
                                 DEFAULT ('foo'::"BarBazBaq")
                                 ON UPDATE ('baz':::"With Emojis too üëç"),
                ADD COLUMN "baR" "DefaultDB".üëé."Mixed Case"
                                 DEFAULT ('foo'::"DefaultDB".public."Mixed Case")
                                 ON UPDATE ('baz':::üëé."Mixed Case")

118:
----------------------------------------------------------------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case" DEFAULT ('foo'::"BarBazBaq") ON UPDATE ('baz':::"With Emojis too üëç"),
                ADD COLUMN "baR" "DefaultDB".üëé."Mixed Case"
                                 DEFAULT ('foo'::"DefaultDB".public."Mixed Case")
                                 ON UPDATE ('baz':::üëé."Mixed Case")

149:
-----------------------------------------------------------------------------------------------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case" DEFAULT ('foo'::"BarBazBaq") ON UPDATE ('baz':::"With Emojis too üëç"),
                ADD COLUMN "baR" "DefaultDB".üëé."Mixed Case" DEFAULT ('foo'::"DefaultDB".public."Mixed Case") ON UPDATE ('baz':::üëé."Mixed Case")

252:
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
ALTER TABLE foo ADD COLUMN "baR" "Mixed Case" DEFAULT ('foo'::"BarBazBaq") ON UPDATE ('baz':::"With Emojis too üëç"), ADD COLUMN "baR" "DefaultDB".üëé."Mixed Case" DEFAULT ('foo'::"DefaultDB".public."Mixed Case") ON UPDATE ('baz':::üëé."Mixed Case")


